homeassistant/components/simplisafe/__init__.py

```diff
@@ -710,11 +712,10 @@ def __init__(
             identifiers={(DOMAIN, serial)},
             manufacturer="SimpliSafe",
             model=model,
-            name=device_name,
+            name=f"{device_name} {' '.join([w.title() for w in model.split('_')])}",
```

Comment:
Shouldn't this be using `capitalize()` instead of `title()`?

Reply:
I'm open. I'm usually more geared toward title-casing things like this (e.g., "Basement Leak Sensor"), but if "Basement leak sensor" is more consistent with what's happening in other integrations, I'm happy to switch.

Reply:
Its part of the new naming: <https://developers.home-assistant.io/docs/core/entity#entity-naming>

> Entity names should start with a capital letter, the rest of the words are lower case (unless it's a proper noun or a capitalized abbreviation of course).

So they all should be sentence-case.

Reply:
@frenck, tell me what you think of 45c8f4eda6ae. Devices names no longer include the model (since that's duplicative with the actual model field):

![CleanShot 2022-07-09 at 11 12 09](https://user-images.githubusercontent.com/47216/178115999-7169c0ff-06e4-44f7-9f9f-855cde9768ca.png)

...and the entities under that device carry names:
![CleanShot 2022-07-09 at 11 13 02](https://user-images.githubusercontent.com/47216/178116019-f571fbf2-d07b-4094-8bd9-d7f80778c1d1.png)

...and entity IDs:

![CleanShot 2022-07-09 at 11 13 48](https://user-images.githubusercontent.com/47216/178116028-c2ed742a-1704-46ae-9e74-71be44a896d8.png)

...consistent with the model. That makes a lot of sense to me, but check me if I'm incorrect.